src/clients/api/queries/useGetVaults/useGetVrtVault.ts

```diff
@@ -0,0 +1,98 @@
+import { useMemo } from 'react';
+
+import { Vault } from 'types';
+import {
+  useGetVrtVaultInterestRatePerBlock,
+  useGetBalanceOf,
+  useGetVrtVaultUserInfo,
+  useGetVrtVaultAccruedInterestWei,
+} from 'clients/api';
+import { DAYS_PER_YEAR } from 'constants/daysPerYear';
+import { BLOCKS_PER_DAY } from 'constants/bsc';
+import { getContractAddress } from 'utilities';
+
+export interface UseGetVrtVaultOutput {
+  isLoading: boolean;
+  data: Vault | undefined;
+}
+
+const VRT_TOKEN_ID = 'vrt';
```

Comment:
This is going to be duplicated throughout the app. I think we  should create a VRT contant file like we have one for XVS. Currently it lives in `pages/ConvertVrt/constants`,

Reply:
Actually, I just updated it to use the `TOKENS` constant directly.
By creating a separate constant for a specific token, we're creating a loop:
```typescript
const TOKENS = {};

const getToken = (tokenId) => TOKENS[tokenId];

const VRT_TOKEN_ID = getToken('vrt').id;
```

When in fact we can just write:
```typescript
const VRT_TOKEN_ID = TOKENS.vrt.id;
```

Reply:
great!